dotcom-rendering/src/web/components/Card/Card.tsx

```diff
@@ -54,6 +55,8 @@ export type Props = {
 	// Labs
 	branding?: Branding;
 	supportingContent?: DCRSupportingContent[];
+	isSnap?: boolean;
+	snapData?: DCREnrichedContent;
```

Comment:
Forgive me if I've already asked this and you've patiently explained why I'm wrong and then I forgot and I'm now asking the same thing again, but would it work to just have `snapData` here and then the decision about if a `Card` is a snap or not is based on the existence of this data?